ui/components/component-library/avatar-account/avatar-account.stories.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import Box from '../../ui/box/box';
 import {
-  ALIGN_ITEMS,
+  AlignItems,
   DISPLAY,
-  SIZES,
+  Size as SizeType,
```

Comment:
Instead of using aliases here I think it's probably better to update the names of the story and use the `storyName` to change the label. The `DefaultStory` is a good example. The name of the story is then changed to `DefaultStory.storyName = 'Default';`

```
export const SizeStory = (args) => (
...
);

SizeStory.storyName = 'Size'
```

Reply:
This is non blocking and we can update but it's more valuable to document with the default enum names in our component stories and docs

Reply:
Thanks for the tip. it is an easy fix with limited files to i followed your suggestion, now it's in the codebase :)